ush/set_extrn_mdl_params.sh

```diff
@@ -162,29 +162,6 @@ fi
 #
 #-----------------------------------------------------------------------
 #
-# Set EXTRN_MDL_LBCS_OFFSET_HRS, which is the number of hours to shift 
-# the starting time of the external model that provides lateral boundary 
-# conditions.
-#
-#-----------------------------------------------------------------------
-#
-case ${EXTRN_MDL_NAME_LBCS} in
-  "GSMGFS")
-    EXTRN_MDL_LBCS_OFFSET_HRS="0"
-    ;;
-  "FV3GFS")
-    EXTRN_MDL_LBCS_OFFSET_HRS="0"
-    ;;
-  "RAP")
-    EXTRN_MDL_LBCS_OFFSET_HRS="3"
-    ;;
-  "HRRR")
-    EXTRN_MDL_LBCS_OFFSET_HRS="0"
-    ;;
-esac
-#
-#-----------------------------------------------------------------------
-#
```

Comment:
Please don't delete this section. It will change the default behavior of the script. It could be useful to have a configurable option to override these settings, though. Maybe if the default value in config_defaults.sh is blank, then these can be used.

Reply:
OK, I added a check. If "EXTRN_MDL_LBCS_OFFSET_HRS" is empty, it will use those default values. Otherwise, use the value set in config.sh.